README.md

```diff
@@ -2,3 +2,136 @@
+exceeding the ABC metric limit, but it's not yet practical to extract business
+logic or presentation into objects.
+
+`.rubocop_todo.yml` should be avoided for more permanent exceptions because it
+generally excludes the whole file from a particular rule, when actually all we
+want is to exclude a single line or single function.
+
+### Scenario 2: a genuine exception
+
+There will inevitably be cases where it is not possible to stick to a rule
+without materially making the code more difficult to understand. Examples
+include:
+
+- Long regular expressions (violating `Metrics/LineLength`)
+- Long case statements (violating `Metrics/MethodLength`
+
+In most cases these should be excluded inline - e.g.
+
+``` ruby
+# rubocop:disable Metrics/LineLength
+  postcode =~ /^\s*((GIR\s*0AA)|((([A-PR-UWYZ][0-9]{1,2})|(([A-PR-UWYZ][A-HK-Y][0-9]{1,2})|(([A-PR-UWYZ][0-9][A-HJKSTUW])|([A-PR-UWYZ][A-HK-Y][0-9][ABEHMNPRVWXY]))))\s*[0-9][ABD-HJLNP-UW-Z]{2}))\s*$/i
+# rubocop:enable Metrics/LineLength
+```
+
+This has the downside of adding noise to into the application code, but the
+advantage that we don't exclude the whole file from the rule, and the risk of
+rubocop exceptions persisting after the issue has been resolved (e.g. by code
+being deleted) is lower)
+
+### Scenario 3: a false positive, or a Rubocop bug
+
+Sometimes Rubocop will mistakenly apply a rule to some irrelevant code - this
+is more of a problem with newer cops. In this case the developer should take a
+view on whether to handle this as a temporary exclusion (and generate a todo
+file) or as a permanent one (and exclude with an inline comment).
+
+### Scenario 4: other people's code
+
+Any kind of automatically generated code, or code copied from another project
+which we don't expect to edit ourselves should not be in the scope of our
+rubocop checks.
+
+A common example is config files for gems like rollbar which live in
+`config/intializers`.
+
+These can be excluded from specific checks in the main `.rubocop.yml` file, or
+if there are multiple failures they can be excluded from all cops.
+
+### Scenario 5: DSLs
+
+Ruby DSLs often make use of `do...end` blocks and so are likely to violate
+`Metrics/BlockLength` and possibly `Metrics/BlockNesting`. In our default
+`.rubocop.yml` we exclude `config/routes.rb` and all RSpec tests from these
+checks. Any additional files written in a DSL should be similarly ignored.
+
+## Adding Rubocop to a legacy project
+When applying rubocop to an existing project we're unlikely to be able to
+address all the existing issues in one go - the important thing is to
+establish standards going forwards.
+
+### 1. Autofix all autofixable issues
+
+Running `rubocop -a` will attempt to automatically fix most style violations.
+These changes should be carefully checked: sometimes where the previous
+formatting was particularly funky, autofixes can change the meaning of the
+code.
+
+For ease of review, this should be committed on it's own: it's all
```

Comment:
This is a good rule and worth making explicit, as you've done here

Reply:
Spare `'` on "its"

Reply:
fixed inline